python3 python/main.py --profile resnet50-tf --model "/home/runner/MLC/repos/local/cache/download-file_ml-model-resnet_72dd4fbf/resnet50_v1.pb" --dataset-path /home/runner/MLC/repos/local/cache/get-preprocessed-dataset-imagenet_57cc9c52 --output "/home/runner/MLC/repos/local/cache/get-mlperf-inference-results-dir_477cefa0/test_results/gh_ubuntu-latest x86-reference-cpu-tf-v2.20.0-default_config/resnet50/offline/accuracy" --scenario Offline --count 500 --threads 4 --user_conf /home/runner/MLC/repos/GATEOverflow@mlperf-automations/script/generate-mlperf-inference-user-conf/tmp/b514b7a73da2445aa05c8456650c7c9f.conf --accuracy --use_preprocessed_dataset --cache_dir /home/runner/MLC/repos/local/cache/get-preprocessed-dataset-imagenet_57cc9c52 --dataset-list /home/runner/MLC/repos/local/cache/extract-file_imagenet-aux-da_a458fe4d/val.txt
INFO:main:Namespace(dataset='imagenet', dataset_path='/home/runner/MLC/repos/local/cache/get-preprocessed-dataset-imagenet_57cc9c52', dataset_list='/home/runner/MLC/repos/local/cache/extract-file_imagenet-aux-da_a458fe4d/val.txt', data_format=None, profile='resnet50-tf', scenario='Offline', max_batchsize=32, model='/home/runner/MLC/repos/local/cache/download-file_ml-model-resnet_72dd4fbf/resnet50_v1.pb', output='/home/runner/MLC/repos/local/cache/get-mlperf-inference-results-dir_477cefa0/test_results/gh_ubuntu-latest x86-reference-cpu-tf-v2.20.0-default_config/resnet50/offline/accuracy', inputs=['input_tensor:0'], outputs=['ArgMax:0'], backend='tensorflow', device=None, model_name='resnet50', threads=4, qps=None, cache=0, cache_dir='/home/runner/MLC/repos/local/cache/get-preprocessed-dataset-imagenet_57cc9c52', preprocessed_dir=None, use_preprocessed_dataset=True, accuracy=True, find_peak_performance=False, debug=False, user_conf='/home/runner/MLC/repos/GATEOverflow@mlperf-automations/script/generate-mlperf-inference-user-conf/tmp/b514b7a73da2445aa05c8456650c7c9f.conf', audit_conf='audit.config', time=None, count=500, performance_sample_count=None, max_latency=None, samples_per_query=8)
2025-09-12 23:51:18.686945: I external/local_xla/xla/tsl/cuda/cudart_stub.cc:31] Could not find cuda drivers on your machine, GPU will not be used.
2025-09-12 23:51:18.733437: I tensorflow/core/platform/cpu_feature_guard.cc:210] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2025-09-12 23:51:20.015167: I external/local_xla/xla/tsl/cuda/cudart_stub.cc:31] Could not find cuda drivers on your machine, GPU will not be used.
INFO:imagenet:Loading 500 preprocessed images using 4 threads
INFO:imagenet:loaded 500 images, cache=0, already_preprocessed=True, took=0.0sec
WARNING:tensorflow:From /home/runner/MLC/repos/local/cache/get-git-repo_inference-src_07585a1f/inference/vision/classification_and_detection/python/backend_tf.py:55: FastGFile.__init__ (from tensorflow.python.platform.gfile) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.gfile.GFile.
WARNING:tensorflow:From /opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/tensorflow/python/tools/strip_unused_lib.py:84: extract_sub_graph (from tensorflow.python.framework.graph_util_impl) is deprecated and will be removed in a future version.
Instructions for updating:
This API was designed for TensorFlow v1. See https://www.tensorflow.org/guide/migrate for instructions on how to migrate your code to TensorFlow v2.
WARNING:tensorflow:From /opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/tensorflow/python/tools/optimize_for_inference_lib.py:138: remove_training_nodes (from tensorflow.python.framework.graph_util_impl) is deprecated and will be removed in a future version.
Instructions for updating:
This API was designed for TensorFlow v1. See https://www.tensorflow.org/guide/migrate for instructions on how to migrate your code to TensorFlow v2.
2025-09-12 23:52:16.289691: E external/local_xla/xla/stream_executor/cuda/cuda_platform.cc:51] failed call to cuInit: INTERNAL: CUDA error: Failed call to cuInit: UNKNOWN ERROR (303)
WARNING: All log messages before absl::InitializeLog() is called are written to STDERR
I0000 00:00:1757721136.375368    7826 mlir_graph_optimization_pass.cc:437] MLIR V1 optimization pass is not enabled
INFO:main:starting TestScenario.Offline
TestScenario.Offline qps=0.67, mean=14.9907, time=23.789, acc=76.000%, queries=16, tiles=50.0:15.0888,80.0:22.7311,90.0:23.5158,95.0:23.6375,99.0:23.6879,99.9:23.6992
